packages/project-access/test/file/file-access.test.ts

```diff
@@ -90,4 +90,30 @@ describe('fileAccess', () => {
             expect(exists).toBe(true);
         });
     });
+
+    describe('updatePackageJSON', () => {
+        test('Should update package.json using previous indentation', async () => {
```

Comment:
Can we please add some tests for files with different indenting and different characters like spaces or `\t`?